public/components/tweets/tweets.controller.js

```diff
@@ -1,41 +1,32 @@
 'use strict';
 
 angular.module('textAnalysis')
-	.controller('tweetCtrl', ['$routeParams','$scope','$log', '$location', 'twitterService', 
-		function($routeParams, $scope, $log, $location, twitterService) {
-			//sets the twitter handle display to be equal to the url handle
-
-            $scope.twitterHandle = $routeParams.handle;
+    .controller('tweetCtrl', ['$routeParams', '$scope', '$log', '$location', 'twitterService',
+        function($routeParams, $scope, $log, $location, twitterService) {
 
             //show loading animation until data from twitter is populated
 
             $scope.loadingTweets = true;
-            
-            $scope.words = [];
 
-            /* get sentiment data about text via ng-click, 
-            sending text to the aylien endpoint */
+            /* get analysis data about text via ng-click, 
+            changing location to /results/ where aylien endpoint is called */
 
             $scope.loadResults = function(tweetText) {
                 twitterService.resultsTweet = tweetText;
                 $location.path('/results/' + $routeParams.handle);
             };
 
-            //display sentiment data after returned to an object 
-            //in the twitterService
-
-            $scope.sentimentData = twitterService.sentimentData;
-
             //load tweets after passing the desired screen_name
             //to the twitterService and then to the twitter endpoint.
 
             twitterService.loadTweets($routeParams.handle)
                 .then(function(data) {
                     $log.log(data);
+                    $scope.twitterHandle = data[0].user.screen_name;
```

Comment:
I'm going to merge this.  After our session, I think you know what needs to be done to properly handle an error here.  I'd like to see that implemented in the next PR.